energyplus/EnergyPlus.cc

```diff
@@ -0,0 +1,11 @@
+#include "EnergyPlus.hh"
+#include "Model.hh"
+
+int main() {
+  MyPlantModel modelInstance(0.0, 1000, 300);
+  PlantLocation modelLocation(1, 1, 1, 1);
+  for (int timeStepNum = 1; timeStepNum < 24; timeStepNum++) {
+    modelInstance.simulate(modelLocation);
+    modelInstance.printResult();
+  }
```

Comment:
This is EnergyPlus 10000000x.  It just steps through 24 time steps, calls the model, and leaves.